config/openshift/kustomization.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../default
+- privileged-role-binding.yaml
+patches:
+- patch: |-
```

Comment:
can we generalize name and namepsace like below ?
```
  name: controller-manager
  namespace: system
```

Reply:
done